crates/common/fmt/src/ui.rs

```diff
@@ -306,10 +306,143 @@ impl UIfmt for OtherFields {
     }
 }
 
+impl UIfmt for AccessListItem {
+    fn pretty(&self) -> String {
+        self.storage_keys.pretty()
+    }
+}
+
 impl UIfmt for Transaction {
     fn pretty(&self) -> String {
-        format!(
-            "
+        match self.transaction_type {
+            Some(3) =>
```

Comment:
could we order these asc?

Reply:
Updated in https://github.com/foundry-rs/foundry/commit/e06f98cc043b4fe2c8a47ac9dafedf272176c866